jobs/integr8ly/installation-pipeline.yaml

```diff
@@ -53,20 +50,24 @@
 
         import hudson.plugins.sshslaves.verifiers.*
 
-        String bastionLabel = "${BASTION_URL}-slave"
+        String[] intlyCompTags = ["rhsso_operator_release_tag", 
+                                  "msbroker_release_tag", 
+                                  "gitea_operator_release_tag",
+                                  "webapp_version",
+                                  "webapp_operator_release_tag",
+                                  "middleware_monitoring_operator_version"]
```

Comment:
This has been renamed to `middleware_monitoring_operator_release_tag`

Reply:
I would also add pathing the `backup_version`. Currently, it is set to `master` but this can be changed any time.